diff --git a/node_modules/axios-taro-adapter/src/adapters/taro.js b/node_modules/axios-taro-adapter/src/adapters/taro.js
index 85c8a6a..dac73de 100644
--- a/node_modules/axios-taro-adapter/src/adapters/taro.js
+++ b/node_modules/axios-taro-adapter/src/adapters/taro.js
@@ -9,10 +9,32 @@ function settle(resolve, reject, res, failed){
 }
 
 export default function taroAdapter(config) {
-  return new Promise((resolve,reject)=>{
+  return new Promise((resolve, reject) => {
+    let query = ""
+    const params = config.params ?? {}
+    const paramsArr=[]
+    for (const k in params) {
+      if (params[k] instanceof Array) {
+        params[k].forEach(item=> {
+          paramsArr.push(k+"="+JSON.stringify(item))
+        });
+      } else {
+        if (params[k] !== undefined && params[k] !== null) {
+          console.log('保留', k);
+          paramsArr.push(k + "=" + JSON.stringify(params[k]))
+        } else {
+          console.log('跳过', k);
+        }
+      }
+    }
+    console.log('paramsArr',paramsArr);
+    if (paramsArr.length > 0) {
+      query="?"+paramsArr.join('&')
+    }
+    console.log('query',query);
     Taro.request({
       ...config,
-      url: config.baseURL + config.url,
+      url: config.baseURL + config.url + query,
       data: config.data,
       method: config.method,
       header: config.headers,
